doc/interfaces/BUAN_flow.rst

```diff
@@ -68,18 +72,49 @@ run following workflow::
 
     dipy_buan_shapes subjects/ --out_dir "sm_plots"
 
+--------------------------------------
+Reproducing results on larger dataset:
+--------------------------------------
+
+Complete dataset DIPY Processed Parkinson's Progression Markers Initiative (PPMI)
+Data Derivatives. It can be downloaded from the link below::
+
+     insert link when gets publishes
```

Comment:
Does it mean we should wait for the publication before merging this PR?

Reply:
This is for large data set.
I have published it from my side. waiting for NIH to publish.

Reply:
ok